plugin/cli.go

```diff
@@ -33,44 +35,54 @@ func getSignedJwt(role string, m map[string]string) (string, error) {
 		serviceAccount = credentials.ClientEmail
 	}
 	if serviceAccount == "" {
+		// Check if the metadata server is available.
+		if metadata.OnGCE() {
+			metadataClient := metadata.NewClient(&http.Client{})
+			path := "instance/service-accounts/default/identity?audience=http%3A%2F%2Fvault%2F" + role + "&format=full"
```

Comment:
I think we need a bit more rigor around building this path. In particular, the query escaping might get weird if the role name has special characters. It would probably be safer to build a [`url.Values`](https://pkg.go.dev/net/url#Values) instead.

Reply:
Done